src/features/accuse/index.tsx

```diff
@@ -41,9 +41,10 @@ export function Accuse() {
     // if there are no archaeologist private keys left, add an empty one
     if (newArchaeologistPrivateKeys.length === 0) {
       newArchaeologistPrivateKeys.push('');
+      setArchaeologistPrivateKeys(['']);
```

Comment:
Removing the last private key field left over the last valid signature. This makes sure it's cleared so that prepare write would throw an error (error flag renamed to `mayFail`)